.github/workflows/test.yml

```diff
@@ -33,10 +33,10 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macOS-latest, windows-latest]
-        go_version: [1.17.x, 1.18.x, 1.19.x]
+        go_version: [1.17.x, 1.18.x, 1.19.x, 1.20.x]
```

Comment:
i think you can remove `1.17` and maybe also `1.18` as i think it would be fine to support 2 recent versions only

Reply:
Okay changed.